c2_components/include/mfx_c2_encoder_component.h

```diff
@@ -202,6 +210,10 @@ class MfxC2EncoderComponent : public MfxC2Component
     MfxC2VppWrapp m_vpp;
     MfxC2Conversion m_inputVppType;
 
+    mfxExtVideoSignalInfo m_signalInfo;
+    std::shared_ptr<C2StreamColorAspectsInfo::input> mColorAspects;
```

Comment:
Please follow the naming style like m_xxx

Reply:
Updated.